korman/exporter/etlight.py

```diff
@@ -375,6 +375,8 @@ def _prep_for_lightmap(self, bo, toggle):
         # this tended to create sharp edges. There was already a discussion about this on the
```

Comment:
(Commenting here, because Github won't let me comment further up)

Small fix in case the user wants to lightmap a textureless object:
```
# Because the way Blender tracks active UV layers is massively stupid...
if uv_textures.active:
    self._uvtexs[mesh.name] = uv_textures.active.name
```